### DISCUSSION TITLE
Material de la primera misión?

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#2] [Q&A] [answered]
Hola compañeros, estoy tratando de ver el material de mañana, pero no encuentro donde está, ya estará disponible?

**Comment by AzaelBarbosa** [comment]
Hola, no hay material, la misión es leer el INTRO para el día de mañana

**Reply by Nocturnal-OFC** [reply]
Gracias por la aclaración

**Reply by AzaelBarbosa** [reply]
De nada companero :)

**Comment by doguedogue** [comment, marked as answer]
Leer todo este material

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO

![image](https://user-images.githubusercontent.com/23409026/154182528-44616a00-f763-4d3e-af1c-c0996f418366.png)

**Reply by MisaelFloresB** [reply]
Estructura de código aun no esta disponible ¿cierto?

**Reply by Nocturnal-OFC** [reply]
En efecto, aún no está disponible

**Reply by MauricioNovelo** [reply]
Gracias viejo, eres como un guru
